backend/services/music_gen_service/music_researcher.py

```
@@ -0,0 +1,225 @@
+"""
+Simple music research service that uses Perplexity to enhance music descriptions.
+Uses OpenAI client library to connect to Perplexity's API.
+"""
+import os
+import logging
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)